src/components/common/YourCompany/OfficeAddresses.vue

```diff
@@ -35,76 +37,78 @@
+                        <span class="ml-1">{{ editLabel }}</span>
+                      </v-list-item-subtitle>
+                    </v-list-item>
+                  </v-list>
+                </v-menu>
+              </span>
+            </div>
+          </v-col>
 
-        <v-flex xs1 v-if="isCorrectionFiling && officeAddressesChanged">
-          <div class="actions mr-4">
-            <span class="edit-action">
+          <v-col cols="1" v-else-if="isCorrectionFiling || isChangeFiling">
+            <div class="actions mr-4">
               <v-btn
                 text color="primary"
-                id="btn-undo-office-addresses"
-                @click="resetOfficeAddresses(); dropdown = false"
+                id="btn-correct-office-addresses"
+                @click="isEditing = true"
               >
-                <v-icon small>mdi-undo</v-icon>
-                <span>Undo</span>
+                <v-icon small>mdi-pencil</v-icon>
+                <span>{{ editLabel }}</span>
               </v-btn>
-            </span>
-            <span class="more-actions">
-              <v-menu
-                offset-y left nudge-bottom="4"
-                v-model="dropdown"
-                attach="#office-addresses .more-actions"
-              >
-                <template v-slot:activator="{ on }">
-                  <v-btn
-                    text small color="primary"
-                    id="btn-more-actions"
-                    v-on="on"
-                  >
-                    <v-icon>{{dropdown ? 'mdi-menu-up' : 'mdi-menu-down'}}</v-icon>
-                  </v-btn>
-                </template>
-                <v-list>
-                  <v-list-item
-                    class="v-list-item"
-                    id="btn-more-actions-edit"
-                    @click="isEditing = true; dropdown = false"
-                  >
-                    <v-list-item-subtitle>
-                      <v-icon small>mdi-pencil</v-icon>
-                      <span class="ml-1">Correct</span>
-                    </v-list-item-subtitle>
-                  </v-list-item>
-                </v-list>
-              </v-menu>
-            </span>
-          </div>
-        </v-flex>
-
-        <v-flex xs1 v-else-if="isCorrectionFiling">
-          <div class="actions mr-4">
-            <v-btn
-              text color="primary"
-              id="btn-correct-office-addresses"
-              @click="isEditing = true"
-            >
-              <v-icon small>mdi-pencil</v-icon>
-              <span>Correct</span>
-            </v-btn>
-          </div>
-        </v-flex>
-      </v-layout>
+            </div>
+          </v-col>
+        </template>
+      </v-row>
 
-      <v-layout row id="summary-records-address" class="mt-4 mx-0" v-if="entityFilter(CorpTypeCd.BENEFIT_COMPANY)">
-        <v-flex xs3>
+      <v-row v-if="entityFilter(CorpTypeCd.BENEFIT_COMPANY)" id="summary-records-address" class="mt-4 mx-0" no-gutters>
```

Comment:
I replaced some `entityFilter(CorpTypeCd.BENEFIT_COMPANY)` with `isBComp` for cleaner code. What do you think here?

Reply:
Definitely! Will update

Reply:
updated